fe1-web/features/events/objects/LaoEvent.ts

```diff
@@ -1,12 +1,17 @@
 import { Hash, Timestamp } from 'model/objects';
 
+/**
+ * Interface to represent an event within a LAO.
+ */
+
+// Represents all types of events
 export enum LaoEventType {
   MEETING = 'MEETING',
   ROLL_CALL = 'ROLL_CALL',
   ELECTION = 'ELECTION',
-  // Election, Poll, etc.
 }
 
+// Serializable LaoEvent (using primitive types)
 export interface LaoEventState {
```

Comment:
FYI, the beauty of these interfaces is that other things (like RollCall and Meeting) can match them *without* importing them :)

Reply:
Okay, that's nice ! I didn't know that.